src/main/java/resources/config.example

```diff
@@ -0,0 +1,3 @@
+*Please create a config.properties file (default path: src/main/java/resources/config.properties) and define paths to both webdrivers as shown below:
+CHROME_DRIVER = please/put/your/path/to/chromedriver.exe
+FIREFOX_DRIVER = please/put/your/path/to/geckodriver.exe
```

Comment:
There are more parameters on master branch on config example in the other catalog - could you make sure that it will be cleaned up in this pr?